backend/Dockerfile

```
@@ -0,0 +1,13 @@
+FROM python:latest
```

```suggestion
FROM python:3.9
```
We have not upgraded to python 3.10 here so it would be best if the docker image is based on python 3.9